src/Http/Controllers/AssetController.php

```diff
@@ -95,7 +95,7 @@ public function manifest(): JsonResponse
 
     public function pwaServiceWorker(): Response
     {
-        return response(Vite::content('resources/js/sw.js', 'flux/build'))
+        return response(Vite::content('resources/js/sw.js', flux_path('public/build')))
```

Comment:
I am just curious if this is correct. Because there may not be a public/build path in flux folder, or am I wrong about this?

Reply:
should be public path not flux path because the assets now live in the public/build of the project.